src/hotspot/share/gc/g1/heapRegion.hpp

```diff
@@ -524,9 +520,9 @@ class HeapRegion : public CHeapObj<mtGC> {
 
   void record_surv_words_in_group(size_t words_survived);
 
-  // Determine if an object is in the parsable or the to-be-scrubbed area.
-  inline static bool obj_in_parsable_area(const HeapWord* addr, HeapWord* pb);
-  inline static bool obj_in_unparsable_area(oop obj, HeapWord* pb);
+  // Determine if an address is in the parsable or the to-be-scrubbed area.
+  inline        bool is_in_parsable_area(const void* const addr) const;
```

Comment:
spacing to align method names seems new to me.

Reply:
There are some in `g1FullCollector.hpp`. I think this alignment groups the identical part while highlights the diff.